CONTRIBUTING.md

```diff
@@ -5,24 +5,27 @@ and the opportunity to contribute to their developer tools. Wrangler is also a p
 delivered by Cloudflare, so it's important to clarify how we think about issue triage and
 contributions.
 
+If you want to learn about how to get started with Wrangler, [click here](#Developing-Wrangler)
+
 ## People
 
-Wrangler is maintained by @ashleygwilliams, and her team, Workers Developer Experience.
+Wrangler is maintained by the Workers Developer Productivity team. The core maintainers are @EverlastingBugstopper and @ashleymichal, and everything that gets merged must be approved by at least one team member.
```

Comment:
I'd suggest just saying the "Workers" team. It has the benefit of being both true and more inclusive; I also like how it helps abstract the most-relevant concept of "core maintainers" away from Cloudflare's organizational structure.

Reply:
I'm not sure it's more inclusive. What about just maintained by Cloudflare?

Reply:
could say `owned by the Cloudflare Workers team and maintained by anybody who wants to contribute`